frontend/components/customizer/query-string-builder.js

```diff
@@ -53,10 +53,10 @@ export default class QueryStringBuilder extends React.Component {
       badgeOptions[name] = defaults[name] || ''
     })
 
-    this.state = { queryParams, badgeOptions }
+    this.state = { queryParams, badgeOptions, defaultStyle }
```

Comment:
Since this is effectively fixed for an instance of the component, it's unnecessary to copy it from `props` to `state`. It also requires passing it around through these function calls. Unless it complicates testing for some reason, what I'd suggest is to change `getQueryString` from a static method to an instance method, and then pull `defaultStyle` out of props within `getQueryString`. That should avoid confusion around why that value is in the state, and also reduce the amount of code needed to copy the value around. Does that make sense?

Reply:
Yeah - that sounds sensible. I'll have a look at re-organising it later or tomorrow